network/p2p/inspector/validation/control_message_validation_inspector.go

```diff
@@ -365,10 +410,11 @@ func (c *ControlMsgValidationInspector) inspectIWantMessages(from peer.ID, iWant
 // - activeClusterIDS: the list of active cluster ids.
 // Returns:
 // - InvalidRpcPublishMessagesErr: if the amount of invalid messages exceeds the configured RPCMessageErrorThreshold.
-func (c *ControlMsgValidationInspector) inspectRpcPublishMessages(from peer.ID, messages []*pubsub_pb.Message, activeClusterIDS flow.ChainIDList) error {
+// - int: the number of invalid pubsub messages
+func (c *ControlMsgValidationInspector) inspectRpcPublishMessages(from peer.ID, messages []*pubsub_pb.Message, activeClusterIDS flow.ChainIDList) (error, uint64) {
```

Comment:
This and some other methods of the `ControlMsgValidationInspector` do not document the returned errors, and more importantly, whether the errors are irrecoverable or benign. Please document the errors and their type on missing places.

Reply:
https://github.com/onflow/flow-go/pull/4773/commits/de1c130ea1951bb4e3dc1c84aeee7ddeb34b2eba